catalog/task/collect-data/0.3/README.md

```diff
@@ -18,5 +25,9 @@ This task also stores the passed resources as json files in a workspace.
 | snapshot             | Namespaced name of the Snapshot                    | No       | -             |
 | subdirectory         | Subdirectory inside the workspace to be used.      | Yes      | -             |
 
+## Changes since 0.2
+  * Added task results for the path to each of the stored resources and one to the subdirectory containing
```

Comment:
subdirectory is no longer included in the results

Reply:
fixed, thanks